src/utils/gitignore.js

```diff
@@ -92,7 +92,7 @@ const ensureNetlifyIgnore = async function (dir) {
     // ignore
   }
   /* Not ignoring .netlify folder. Add to .gitignore */
-  if (!ignorePatterns || !ignorePatterns.patterns.includes('.netlify')) {
+  if (!ignorePatterns || !ignorePatterns.patterns.some((pattern) => pattern.includes('.netlify'))) {
```

Comment:
Do you think there's a risk that someone uses sub-directories like `mysite.netlify.app`, which we'd be targeting with this new logic and incorrectly skipping the `.gitignore` modification? It's probably an edge case, but I wonder if a regular expression like `(^|\/)\.netlify($|\/)` would be more accurate?

Reply:
Great point. Done in https://github.com/netlify/cli/pull/2031/commits/38497c1640b31f87f7380551b07ea5791e2386ce

Reply:
Sorry, I just realised that this might not account for Windows paths? Perhaps `(^|\/|\\)\.netlify($|\/|\\)` is better?

Reply:
Nice one, done in https://github.com/netlify/cli/pull/2031/commits/cff8f941ba917fc8ba3c514d71d5e18d2af6bd1d